include/roboteam_ai/stp/Tactic.h

```diff
@@ -7,18 +7,14 @@
 
 #include <vector>
 #include <roboteam_utils/containers/state_machine.hpp>
-
 #include "stp/StpInfo.h"
-#include "stp/new_skills/Rotate.h"
-#include "stp/new_skills/SetDribbler.h"
+#include "stp/Skill.h"
 
 namespace rtt::ai::stp {
-    class Skill;
+class Skill;
 
-    class Tactic {
+class Tactic {
     private:
```

Comment:
Should be protected - you want to inherit and use these functions

```suggestion
    protected:
```